test/clj_http/test/core_test.clj

```diff
@@ -104,6 +104,8 @@
     {:status 200 :body "delete-with-body"}
     [:post "/multipart"]
     {:status 200 :body (:body req)}
+    [:head "/head-with-body"]
+    {:status 200 :headers {"body" (slurp (:body req))}}
```

Comment:
This is weird, but I'm not seeing a better way to test this in this framework. I'm just putting the body back into a header, to confirm that it was received, since the body in the response should be empty.